notes/1.3/notes.MD

```diff
@@ -1,4 +1,4 @@
-Advance react component patterns is a good course that covers in detail what design patterns are and how to apply them to react.
+Advanced react component patterns is a good course that covers in detail what design patterns are and how to apply them to react.
```

Comment:
These notes are very limited; please include information and code snippets about all the different patterns learned from this course